weblate/trans/vcs.py

```diff
@@ -630,6 +631,75 @@ def push(self, branch):
 
 
 @register_vcs
+class GithubRepository(GitRepository):
+
+    name = 'GitHub'
+
+    _cmd = 'hub'
+    _hub_config_file = os.path.join(os.path.expanduser('~'),
+                                    '.config/hub')
```

Comment:
Placing this to DATA_DIR might be better approach, as user home is not really usable location for most of hostings.